### DISCUSSION TITLE
Were some tab attributes removed in Firefox v110.0?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [mdn/mdn-community#366] [Content] [answered]
Hello, well, I want to say, that some tab attributes were removed in the Firefox v110.0, but why these attributes weren't written in the developer release notes? (here: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/110)

------------------------------------

I'm quoting Mr. black7375:
[Many attributes has been deleted](https://bugzilla.mozilla.org/show_bug.cgi?id=1808661), and the code must be rewritten.
In some cases, it may be difficult to implement.

    beforeselected-visible https://github.com/mozilla/gecko-dev/commit/968a0377089083adb5f622ba722e0055b678d8c7 https://github.com/mozilla/gecko-dev/commit/8ec7bf5a6a2a431445f7d2bf46a2fb57b7396fa9
    first-visible-tab https://github.com/mozilla/gecko-dev/commit/f46a7d03a60748d2e6d7afdd0229e46beefb51c0 https://github.com/mozilla/gecko-dev/commit/ab88c0d61de139ad088648f48f4661660b2974ca
    last-visible-tab https://github.com/mozilla/gecko-dev/commit/0fea6a7914f2d73b60d5e09fd10937d6405b02ea https://github.com/mozilla/gecko-dev/commit/e4262236d5da3dfb14feccc37326c9cf7334fb6b
    before-multiselected https://github.com/mozilla/gecko-dev/commit/a1eb3c4eb8b7466c8da9f4d2586a277831d96aa4
    beforehovered & afterhovered https://github.com/mozilla/gecko-dev/commit/14f328f107bda75de75585dcf34a7d11df8b7fdd https://github.com/mozilla/gecko-dev/commit/849bfbb19b7b033552fb4fe5ac992246c07da3e0

------------------------------------

Well, he said, that these attributes were removed, but now the most important question: Why did you haven't written it in the mentioned MDN release notes page? Did someone forget? Or it is an intentional decision (E.g. We didn't wrote it here, because of this reason and this, and…)

No offence, I'm just thinking out loud.

**Comment by dipikabh** [comment, marked as answer]
Hello @Patricek0056, this seems like a UI-specific issue and a better place to capture this change would probably be https://www.mozilla.org/en-US/firefox/110.0/releasenotes/.
